src/classes/Teacher.java

```diff
@@ -1,28 +1,21 @@
 package classes;
 
-public class Teacher {
-    // <editor-fold desc="Attributes">
-    private String name;
+public class Teacher extends Employee{
+
     private int age;
     private int employmentYears;
-    // </editor-fold>
 
-    // <editor-fold desc="Constructor">
     public Teacher(String name, int age, int employmentYears) {
-        this.name = name;
+            super(name, 0);
+            this.age = age;
+            this.employmentYears = employmentYears;
+        }
+
+    public Teacher(String name, int age, int timeWork, double salary) {
+        super(name, salary);
         this.age = age;
         this.employmentYears = employmentYears;
```

Comment:
Percebi um erro aqui que o parâmetro do construtor ainda está com nome timeWork e aqui está sendo atribuido employmentYears;

Como o nome do parâmetro está timeWork e não employmentYears, esta linha está essencialment fazendo isso:
`this.employmentYears = this.employmentYears`

Ou seja, reatribuindo o valor da variável nela mesma. O valor passado como parâmetro timeWork nunca é utilizado.
Dá pra confirmar isso na classe Main pois o objeto teacher fica com employmentYears 0 mesmo sendo passado 10 no construtor.

@lucascarnu, eu vou aceitar o pull request só pra você não ter que fazer fazer ele uma terceira vez. Aí eu vou criar o meu próprio pull request corrigindo esse bug, e você revisa e faz o merge, pode ser?